app/views/layouts/standard.html.erb

```diff
@@ -25,12 +25,14 @@
               <div class="users-link-container">
                 <%= link_to "Users", users_path, class: "users-link" %>
               </div>
-                  <header>
+                <header>
                     <% if current_user %>
                       <%= link_to "Sing out", destroy_user_session_path, 
                       'data-turbo-method': :delete %>
+                    <% else %>
```

Comment:
[OPTIONAL BUT RECOMMENDED] Please make sure your sign out functionality is properly working, for now, it gives me this error: `Couldn't find User with 'id'=sign_out` so the solution is to change your `link_to` with `button_to` and also change `data-turbo-method` with `method` because whenever any user clicks on signout button it calls the delete method but if you using a link instead of a button it redirects to the `user_show_path` page and when you use button it calls the delete method and the user will signout so kindly fix this issue as well :slightly_smiling_face: 

![Screenshot from 2024-01-27 21-14-58](https://github.com/yusufyusie/blog_app/assets/45226848/fa07d492-fcc0-4243-a4b2-501be9761472)


```suggestion
                     <%= button_to "Sign out", destroy_user_session_path, 
                      method: :delete %>
                    <% else %>
```